spec/unit/models/user_filters/user_filter_builder_spec.rb

```diff
@@ -107,5 +107,16 @@
         end
       end
     end
+
+    context 'when creating MUFs results in errors' do
+      before do
+        allow(client).to receive(:post)
+                           .and_return('userFiltersUpdateResult' => {'failed' => [{status: :failed}]})
+      end
+
+      it 'fails' do
+       expect { subject.execute_mufs(filter_definitions, options) }.to raise_error /Creating MUFs resulted in errors/
```

Comment:
Lint/AmbiguousRegexpLiteral: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.